runtime/queries/rust/highlights.scm

```diff
@@ -48,6 +48,55 @@
   "'" @label
   (identifier) @label)
 
+; ---
+; Prelude
+; ---
+
+((identifier) @constant.builtin
```

Comment:
I am not sure if these are really constants. These are normally highlighted as enum variants. I think these prelude highlights should only specialize the highlight scope (like `type.builtin` instead of `type`) not change it entirely.

Reply:
In theory the scoppe is supposed to match here. So I guess it would be `type.enum.variant.builtin`?

I don't see that pattern actually matching for me tough (instead `Ok()` seems to match `constructor`). Maybe there is also a precedance issue? @the-mikedavis I think you know more than me here

Reply:
With the interpretation that these are values at runtime, any type is a constant. As enum members these are certainly types and I would consider neovim's queries to be incorrect. Pascal's suggestion of `type.enum.variant.builtin` is probably the most accurate here although I don't believe any themes define that key yet

Reply:
That's fair, pushed an update with that change